openslides/users/static/templates/users/user-list.html

```diff
@@ -289,7 +290,14 @@ <h1 translate>Participants</h1>
               <a ui-sref="users.user.detail({id: user.id})" class="title">{{ user.get_short_name() }}</a>
             </strong>
           </div>
-          <div ng-if="user.number"><translate>No.</translate> {{ user.number }} </div>
+          <div os-perms="users.can_manage"> <!-- user number -->
+            <div ng-if="user.number" editable-text="user.number" onaftersave="save(user)">
+              <translate>No.</translate> {{ user.number }}
```

Comment:
We need a context or comment string because we want our translators to recognize this string as abbrevation for number, not as the opposite of yes.